_data/header.yml

```diff
@@ -3,12 +3,21 @@
 # Uncomment this statement (by removing the leading '# ') to set the
 # header title differently from the site's title (as defined in
 # _config.yml).
-# title: Header title
+
+title: ""
+
+# Used to show or hide the official usa banner at the top of the site.
+# Comment out to the following line to hide the usa banner.
```

Comment:
or set to `false`? Which is preferred?

Reply:
Either works, I think we chose to comment out as that is the format for most all of the other config options